java/src/jmri/jmrit/dispatcher/TrainInfoFile.java

```diff
@@ -679,6 +679,8 @@ public List<TrainInfoFileSummary> getTrainInfoFileSummaries() {
                 summaries.add(new TrainInfoFileSummary(fileName));
             } catch (IOException ex) {
                 summaries.add(new TrainInfoFileSummary(fileName));
+            } catch (RuntimeException ex) {
+                log.error("Traininfo File [{}] unexplained error, currupted?",fileName);
```

Comment:
No need to change, but I think `currupted` is misspelled. Should be `corrupted`.